### DISCUSSION TITLE
How to Hide Label for Nodes and Edges

### DISCUSSION THREAD

**Discussion by NayeerBM** [antvis/G6#3391] [Q&A] [answered]
Hi. I wanted to ask, how do I hide labels for all nodes and edges when I click on a button? I tried going through the documentation but couldn't seem to find how to hide the labels. Would appreciate it if anyone could help with this.

**Comment by mostopalove** [comment, marked as answer]
Hello. Try to change the label style in label config for all graph elements by clicking

**Reply by mostopalove** [reply]
https://g6.antv.vision/en/docs/manual/middle/elements/methods/updateElement#update--updateitem
